### PR TITLE
Add basic Rust sample

### PR DESCRIPTION
**Pull request by nvanbenschoten**
This sample uses the newly supported Rust driver.

My gift to @tamird and @tschottdorf.

---
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/docs/294)

### CODE REVIEW

File: _includes/app/basic-sample.rs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+    // Print out the balances.
+    println!("Initial balances:");
+    for row in &conn.query("SELECT id, balance FROM accounts", &[]).unwrap() {
+        let id: i64 = row.get(0);
+        let balance: i64 = row.get(1);
+        println!("{} {}", id, balance);
+    }
+}
+
```
Comment (tamird) [inline _includes/app/basic-sample.rs]: random newlinew

**Comment by tamird** [comment]
No placeholders?

LGTM

**Comment by tschottdorf** [comment]
👍 LGTM

**Comment by nvanbenschoten** [comment]
TFTRs. I'll wait until this actually works in production on our end and on the driver's end before merging. 

And we don't use placeholders in any of our basic samples (https://www.cockroachlabs.com/docs/build-a-test-app.html).

**Comment by jseldess** [comment]
Thanks for adding this, @nvanbenschoten. Do you want to add the transaction sample as well?

**Comment by nvanbenschoten** [comment]
@jseldess I have a working transaction example, but it's blocked on a pair of changes that need to be pushed upstream. Until then it doesn't make sense to add it to the docs, but I'll send a PR when it's ready to go.

**Pull request merged**